connector/spanmetricsconnector/connector.go

```diff
@@ -321,6 +338,29 @@ func (p *connectorImp) aggregateMetrics(traces ptrace.Traces) {
 				// aggregate sums metrics
 				s := sums.GetOrCreate(key, attributes)
 				s.Add(1)
+
+				// aggregate events metrics
+				if p.events.Enabled && len(p.events.Dimensions) > 0 {
```

Comment:
Done

Reply:
@albertteoh I have added `validateEventDimensions` function and added tests for that. But one test is failing in `connector_test.go`. The test case is when events are enabled but dimensions list is empty. I think the validate function is not being called in my test `TestSpanMetrics_Events`. Do you have any better way of testing this usecase?

Reply:
Fixed the testcase by assuming that validateConfig would break if event dimensions are empty and removed that particular testcase from connector_test.go since it's already tested in validateDimensions